common/src/main/java/com/wynntils/features/utilities/TranslationFeature.java

```diff
@@ -25,7 +25,7 @@
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 @StartDisabled
-@ConfigCategory(Category.UNCATEGORIZED)
+@ConfigCategory(Category.UTILITIES)
```

Comment:
Slightly more dubious, but ... perhaps. It's not really a "Utility" in the same sense as the others. But then again, this functionality is mostly broken in Artemis anyway right now. If I were to try to get it to work better, I will probably want to reclassify it.